src/ruby/yast/y2base.rb

```diff
@@ -0,0 +1,131 @@
+module Yast
+  module Y2Base
```

Comment:
IMHO having both a program and a module named y2base is confusing. I would accept it if the program was a one-liner, or we should rename the module.